data/vtlib/bcl2bam_phix_deplex_wtsi_stage1_template.json

```diff
@@ -68,7 +68,14 @@
 	{"id":"qc_check_position","required":"yes","default":{"subst":"i2b_lane"}},
 	{"id":"qc_check_qc_in_dir","required":"yes","default":{"subst_constructor":{"vals":[{"subst":"i2b_runfolder_path"}, "Data/Intensities", {"subst":"i2b_bam_basecalls_dir"}],"postproc":{"op":"concat","pad":"/"}}}},
 	{"id":"qc_check_qc_out_dir","required":"yes","default":{"subst_constructor":{"vals":[{"subst":"qc_check_qc_in_dir"}, "no_cal/archive/qc"],"postproc":{"op":"concat","pad":"/"}}}},
-	{"id":"run_lane_label", "description":"label constructed from id_run and position", "default":{"subst_constructor":{"vals":[{"subst":"qc_check_id_run"}, "_", {"subst":"i2b_lane"}],"postproc":{"op":"concat","pad":""}}}}
+	{"id":"run_lane_label", "description":"label constructed from id_run and position", "default":{"subst_constructor":{"vals":[{"subst":"qc_check_id_run"}, "_", {"subst":"i2b_lane"}],"postproc":{"op":"concat","pad":""}}}},
+	{
+		"id":"s1_split_reference_flag",
+		"required":"no",
+		"comment":"flag will disappear unless scramble_reference_fasta value is given (allows unaligned cram)",
```

Comment:
We should be forcing reference based compression off instead of passing a reference.  I understand reference-based CRAM compression is generally a bad thing if the file is not coordinate sorted (which it isn't here right?).